docker-entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -ex
+
+if [[ ! -d "$DATA_DIR" ]]; then
+        exit 1
+fi
+
+chown $UID:$GID /opt/hoover
+chown -R $UID:$GID $DATA_DIR
```

Comment:
Please remove the recursive `chown` from the entrypoint: for large data dirs, this will take a very long time, even if there's no permission to change!